src/login/logind-core.c

```diff
@@ -411,6 +411,10 @@ int manager_get_idle_hint(Manager *m, dual_timestamp *t) {
 
         assert(m);
 
+        /* Initialize the baseline timestamp with the time the manager got initialized to avoid reporting
+         * unreasonable large idle periods starting with the Unix epoch. */
+        ts = m->init_ts;
```

Comment:
maybe drop the DUAL_TIMESTAMP_NULL initialization further up then, given you always override this here...